ui/ui-frontend/projects/archive-search/src/app/archive/archive-search/archive-search.component.ts

```diff
@@ -985,9 +985,11 @@ export class ArchiveSearchComponent implements OnInit, OnChanges, OnDestroy, Aft
           .pipe(filter((result) => !!result))
           .subscribe(() => {}),
       );
-    } else if (this.selectedItemCount <= this.RECLASSIFICATION_THRESHOLD) {
+    } else {
       this.archiveUnitGuidSelected = this.isAllChecked
```

Comment:
this.archiveUnitGuidSelected peut être une constante locale à la place d'un attribut de classe.